rubin_sim/scheduler/basis_functions/feasibility_funcs.py

```diff
@@ -51,6 +52,26 @@ def check_feasibility(self, conditions):
         return result
 
 
+class OnceInNightBasisFunction(BaseBasisFunction):
```

Comment:
I suspect tagging things as "feasibility function that's a boolean" vs. "basis function with a value" and then maybe also as spatial or single value is going to be handy sooner rather than later, just for easier readability.